CONTRIBUTING.md

```diff
@@ -311,7 +311,9 @@ class Foo
   }
 };
 ```
-
+*_NOTE:_* <br/>
+What to do with the usage of the QAC++ suppression syntax in our current code base? <br/>
+We should check them manually and replace them with Axivion syntax.
```

Comment:
@mossmaurice what do you think? Create an issue and remove the QAC++ suppression syntax?

Reply:
@elBoberido Done: https://github.com/eclipse-iceoryx/iceoryx/issues/1566

Reply:
@shankar-in please remove this comment again. It will be taken care of in the issue @mossmaurice created

Reply:
@shankar-in sorry, I meant @saif-at-github :sweat_smile: